bedrock/settings/base.py

```diff
@@ -406,6 +406,19 @@ def lazy_langs():
 SESSION_COOKIE_SECURE = not DEBUG
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
+# legacy setting. backward compat.
+DISABLE_SSL = config('DISABLE_SSL', default=True, cast=bool)
+# SecurityMiddleware settings
+SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', cast=int)
+SECURE_HSTS_INCLUDE_SUBDOMAINS = False
+SECURE_BROWSER_XSS_FILTER = config('SECURE_BROWSER_XSS_FILTER', default=False, cast=bool)
+SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=not DISABLE_SSL, cast=bool)
```

Comment:
The thing is that `SECURE_SSL_REDIRECT` is a the variable that the [django middleware needs](https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-SECURE_SSL_REDIRECT). So we need to use it. 

As for making the secure redirect the default this is once again our discussion about whether to make config defaults for dev or server. I suggest that it'll take a lot more of our time to make all devs change their config than it would to leave it default to non-ssl and adjust our server configs.